packages/create-sitecore-jss/src/templates/nextjs-sxa/src/lib/site-resolver/plugins/default.ts

```diff
@@ -0,0 +1,14 @@
+import { SiteInfo } from '@sitecore-jss/sitecore-jss-nextjs/site';
+import config from 'temp/config';
+import { SiteResolverPlugin } from '..';
+
+class DefaultPlugin implements SiteResolverPlugin {
+  exec(sites: SiteInfo[]): SiteInfo[] {
+    /**
+     * remove default logic. it breaks for getting site info about context site language
```

Comment:
@matkovskyi Think we're still not on the same page 🤔 We shouldn't need to remove default plugin logic. Site resolution/context is working as expected. Let's discuss again.